### PR TITLE
changes "create a job" docs

### PR DESCRIPTION
**Pull request by bcsherma**
## Description
What does the pull request do? If it fixes a bug or resolves a feature request, be sure to link to that issue.

I've removed a lot of the example code that was repeated several times, and instead I am linking to the relevant documentation for the different methods for associating source code with a wandb run. I'm also including a note about the minimum sdk version required to create jobs.

## Checklist
Check if your PR fulfills the following requirements. Put an `X` in the boxes that apply. 

- [x] Files I edited were previewed on my local development server with `yarn start`. My changes did not break the local preview.
- [x] Build (`yarn build`) was run locally and successfully without errors or warnings.
- [x] I merged the latest changes from `main` into my feature branch before submitting this PR.

### CODE REVIEW

File: docs/guides/launch/create-job.md [hunk 1]
```diff
@@ -4,214 +4,28 @@ description: Discover how to create a job for W&B Launch.
 
 # Create a job
 
-A W&B Job is a definition of a computational process. You can think of jobs as a ‘run template’. A job is populated with contextual information about the run it is created from such as the source code, software dependencies, hyperparameter configuration, and more. For more information, see the [View details of launched jobs section](launch-jobs#view-details-of-launched-jobs).
+## What is a job?
```
Comment (KyleGoyette) [inline docs/guides/launch/create-job.md]: Should this be under "Create a Job" or should it be it's own heading?

File: docs/guides/launch/create-job.md [hunk 2]
```diff
@@ -4,214 +4,28 @@ description: Discover how to create a job for W&B Launch.
...
-There are three ways to create a job:
+A job is a complete blueprint of how to perform a step in your ML workflow, like training a model, running an evaluation, or deploying a model to an inference server. For more information, see the [details of launched jobs section](launch-jobs#view-details-of-launched-jobs).
 
-1. Log a code artifact
-2. Associate a Git repository
-3. Set the `WANDB_DOCKER` environment variable
+## How do I create a job?
 
-W&B automatically checks your run to see if it can create a job. W&B determines how to create a job based on the following logic:
+Jobs will be captured automatically from any workloads that you track with W&B if your run has associated source code. You can connect source code to your runs in the following ways:
```
Comment (KyleGoyette) [inline docs/guides/launch/create-job.md:11]: 🔥

File: docs/guides/launch/create-job.md [hunk 3]
```diff
@@ -4,214 +4,28 @@ description: Discover how to create a job for W&B Launch.
...
-![](/images/launch/example_github_repo.png)
-
-The `canonical_job_example.py` script contains this code:
-
-```python
-import random
-import wandb
-
-
-def run_training_run(epochs, lr):
```
Comment (KyleGoyette) [inline docs/guides/launch/create-job.md:101]: Will we no longer provide users with a training script?

File: docs/guides/launch/intro.md [hunk 4]
```diff
@@ -8,32 +8,32 @@ description: Easily queue and manage jobs using W&B Launch.
...
-2. Scroll to the Beta Features sections. Toggle the `W&B Launch` option.
+1. Navigate to https://wandb.ai/settings.
+2. Scroll down to the **Beta Features** section and enable **W&B Launch**.
 
 ![](/images/launch/toggle_beta_flag.png)
 
 ## What is Launch?
 
-Use W&B Launch to start jobs in the compute  environment or infrastructure of choice from one central interface. W&B Launch removes the complexity of using different environments to run training, inference, and more. Weights and Biases will automatically capture the information needed to reproduce the run, containerize the job and launch the job locally or remotely. 
+W&B Launch is a toolkit for building collaborative, portable, and reproducible machine learning workflows.
```
Comment (KyleGoyette) [inline docs/guides/launch/intro.md:20]: Is this repeated from the other PR

File: docs/guides/launch/create-job.md (hunk 1 above)
Comment (ngrayluna) [inline docs/guides/launch/create-job.md]: Should be under "Create a job".


You need text under headings. Can't have a subheading under a title heading.  (looks odd)

File: docs/guides/launch/create-job.md (hunk 3 above)
Comment (ngrayluna) [reply docs/guides/launch/create-job.md:101]: I think we should keep the training script.

File: docs/guides/launch/create-job.md (hunk 2 above)
Comment (ngrayluna) [reply docs/guides/launch/create-job.md:11]: Nit. Active voice re-wording: "Jobs are automatically captured from any workloads you track with W&B if your run has source code associated with it. You can..."

File: docs/guides/launch/create-job.md (hunk 3 above)
Comment (bcsherma) [reply docs/guides/launch/create-job.md:101]: The script appears in a few other places in these docs and I don't see why we are providing a code snippet on this particular page. This isn't a step in a tutorial, it's documentation of how jobs are created.

File: docs/guides/launch/create-job.md (hunk 1 above)
Comment (bcsherma) [reply docs/guides/launch/create-job.md]: I propose just removing the "what is a job?" heading

Comment (bcsherma) [reply docs/guides/launch/create-job.md]: Yup yup

File: docs/guides/launch/intro.md (hunk 4 above)
Comment (bcsherma) [reply docs/guides/launch/intro.md:20]: yass

File: docs/guides/launch/create-job.md (hunk 2 above)
Comment (bcsherma) [reply docs/guides/launch/create-job.md:11]: I think that what you propose changes the tense not the active/passive voicing - is tense also part of the informal style guide?

Comment (bcsherma) [reply docs/guides/launch/create-job.md:11]: I think active would be "W&B automatically captures jobs..."

**Pull request merged**